htdocs/core/lib/functions.lib.php

```diff
@@ -5349,11 +5349,15 @@ function complete_head_from_modules($conf,$langs,$object,&$head,&$h,$type,$mode=
 						if ($values[3]) $langs->load($values[3]);
 						if (preg_match('/SUBSTITUTION_([^_]+)/i',$values[2],$reg))
 						{
-							$substitutionarray=array();
-							complete_substitutions_array($substitutionarray,$langs,$object,array('needforkey'=>$values[2]));
-							$label=make_substitutions($reg[1], $substitutionarray);
+							$substitutionarray=head();
+							complete_substitutions_array($substitutionarray,$langs,$object,array('needforkey'=>$values[2]),'complete_head_from_modules');
```

Comment:
the complete_substitutions_arrayfunction is call in two places : 
- in odt for adding substitution elements (i use this feature on extraodt and customtabs to add some substitution
- in menu for change only the labels of a menu (i plan to use it to reordering tabs)

BUT without parameters it's the same function who is call in substitutions function (in my case extraodt_completesubstitutionarray ) 
with this parameters, I can select the good function to call in my substition file (/core/substitutions/functions_extraodt.lib.php)

With this code i keep the feature of label change but we need to change the function name
from
modulename_completesubstitutionarray 
to  
modulename_complete_head_from_modules
and i enhance it to made more change on menu tabs

I precise that the code is the same, just for the 2 case (5 ou 6 parameters)